glTFLoader/Interface.cs

```diff
@@ -226,6 +228,15 @@ public static Byte[] LoadBinaryBuffer(this Gltf model, int bufferIndex, Func<str
             {
                 throw new InvalidDataException($"The buffer length is {bufferData.Length}, expected {buffer.ByteLength}");
             }
+            if (bufferViewByteOffset > 0)
+            {
+                var newBufferData = new byte[bufferData.Length - bufferViewByteOffset];
+                for (var i = 0; i < newBufferData.Length; i++)
+                {
+                    newBufferData[i] = bufferData[bufferViewByteOffset + i];
+                }

```

Comment:
Probably should use [Array.Copy](https://docs.microsoft.com/en-us/dotnet/api/system.array.copy).